core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-configuration/components/dot-experiments-configuration-goal-select/dot-experiments-configuration-goal-select.component.ts

```diff
@@ -181,27 +145,38 @@ export class DotExperimentsConfigurationGoalSelectComponent implements OnInit, O
                     nonNullable: true,
                     validators: [Validators.required]
                 }),
-                conditions: new FormArray([
-                    new FormGroup({
-                        parameter: new FormControl(''),
-                        operator: new FormControl(''),
-                        value: new FormControl('')
-                    })
-                ])
+                conditions: new FormArray([new FormControl()])
             })
         });
+    }
 
-        this.form.get('primary.type').valueChanges.subscribe((value) => {
-            this.defineDefaultName(value);
-        });
+    private listenType() {
+        this.form
+            .get('primary.type')
+            .valueChanges.pipe(takeUntil(this.destroy$))
+            .subscribe((value) => {
+                this.defineDefaultName(value);
+                this.removeConditionsByType(value);
+            });
+    }
+
+    private removeConditionsByType(typeValue: GOAL_TYPES): void {
+        const conditionsFormArray = this.form.get('primary.conditions') as FormArray;
+        const goalTypeWithoutConditions: GOAL_TYPES[] = [
+            GOAL_TYPES.EXIT_RATE,
+            GOAL_TYPES.BOUNCE_RATE
+        ];
+        if (goalTypeWithoutConditions.includes(typeValue)) {
```

Comment:
why not remove all conditions no matter the goal type to make sure is clear all the time ?

Reply:
Let me try that.

Reply:
Good Idea. Simplified